sympy/physics/mechanics/__init__.py

```diff
@@ -26,7 +26,9 @@
 
     'SymbolicSystem',
 
-    'PinJoint', 'PrismaticJoint'
+    'PinJoint', 'PrismaticJoint',
+
```

Comment:
why the space?

Reply:
Because `PinJoint`, `PrismaticJoint` are imported from different file than `JointsMethod`.